cmd/link-client/main.go

```diff
@@ -150,6 +151,52 @@ func main() {
 				fmt.Println(aurora.Green(fmt.Sprintf("IP %s (%s) successfully added", newIP.IP.IP, newIP.ID)))
 				return nil
 			},
+		}, {
+			Name:      "update",
```

Comment:
I'm not found of the name of this command. Since it does not let a user update all the fields but just the healthchecks.
I would either do:
```
link update-healthcheck [ID] TCP 10.0.0.1:1234
```

Or
```
link update [ID] --healthcheck-type=TCP --healthcheck-port=1234 --healthcheck-host=10.0.0.1
```